### PR TITLE
Add configuration for Synology NAS

### PR DESCRIPTION
**Pull request by nickwaring**

### CODE REVIEW

File: source/getting-started/autostart.markdown [hunk 1]
```diff
@@ -130,6 +132,39 @@ Home Assistant has been uninstalled.
...
 </div> <!-- OSX -->
+
+<div class='advanced-installs synology' markdown='1'>
+To get Home Assistant to automatically start when you boot your Synology NAS:
+
+SSH onto your synology & login as admin or root
+```bash
+$ cd /volume1/homeassistant
+```
+Create "homeassistnat.conf" file using the following code
```
Comment (balloob) [inline source/getting-started/autostart.markdown]: typo

File: source/getting-started/index.markdown [hunk 2]
```diff
@@ -74,6 +74,182 @@ Running these commands will:
...
 </div> <!-- INSTALL-INSTRUCTIONS RASPBERRY -->
+
+<div class='install-instructions synology' markdown='1'>
+
+The following configuration has been tested on a Synology 415+ running DSM 5.2-5644 Update 3
+
+Running these commands will:
+
+ - Install Home Assistant
+ - Ebable Home Assistant to be launched on [http://localhost:8123](http://localhost:8123)
```
Comment (balloob) [inline source/getting-started/index.markdown]: typo

**Comment by nickwaring** [comment]
typos corrected

**Comment by balloob** [comment]
Did you also test it locally to see if it works?

On Mon, Feb 15, 2016 at 12:05 PM, Nick Waring <notifications@github.com>
wrote:

> typos corrected
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/balloob/home-assistant.io/pull/219#issuecomment-184367677>
> .
>



-- 
PaulusSchoutsen.nl
It's nice to be important but it's more important to be nice.

**Comment by nickwaring** [comment]
not yet... I have to upload to my nas and setup web server. I will do it but it will have to wait until later in the week

**Pull request merged**